test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php

```diff
@@ -587,4 +589,45 @@ public function testCaseSensitiveConstant(string $constantName): void
 
         $this->constantReflector->reflect($constantName);
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testUpdateConstantValue(): void
+    {
+        require __DIR__ . '/../../Fixture/FakeConstants.php';
+
+        $sourceStubber = new PhpStormStubsSourceStubber(BetterReflectionSingleton::instance()->phpParser());
+
+        $stubberReflection = new CoreReflectionClass($sourceStubber);
+
+        $stubDirectoryReflection = $stubberReflection->getProperty('stubsDirectory');
```

Comment:
Hacky :P

Reply:
Yes, two hours of creating this test :D

Reply:
I know the feeling - having one of those come up for me too, on a client project :-\